.github/CODEOWNERS

```diff
@@ -104,6 +104,7 @@
 /scripts/__init__.py @kevinlee12
 /scripts/build.py @kevinlee12
 /scripts/build_test.py @kevinlee12
+/scripts/check_e2e_tests_are_captured_in_ci*.py @Showtim3 @kevinlee12
```

Comment:
Please list out the files individually since there aren't too many.

Reply:
@kevinlee12 But if we will start following "list out the files individually" then it will be too many. I think this is the pattern we follow to include the test file (Considering the code owner of the script will also be the code owner for the script's test). I'm fine with listing out files individually but let's finalize one pattern that we will follow in the future. Thanks! :)

Reply:
This only contains the script file and the associated test, which this section already does anyways so that's why I'm advocating for this change. As for the pattern question, I think we're going to have do some surgery to this file since it's slowly getting out of hand.

Reply:
Hm, would a reasonable pattern for the future be: if a codeowner owns all the files in a directory, we just list that directory here? I.e. don't break down unless necessary.

Reply:
Hmm, so I believe it should be kept as it is?

Reply:
sure, leave it as is.